CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ which should look like this:
 ```dotenv
 CDF_CLUSTER=bluefield
 CDF_URL=https://bluefield.cognitedata.com
-CDF_PROJECT=power-ops-staging
+CDF_PROJECT=power-ops-dev

```

Comment:
Here, I would also write power-ops-<your environment> and listing possible choices instead of providing one specific choice.